opennlp-tools/src/main/java/opennlp/tools/util/HashList.java

```diff
@@ -27,27 +26,46 @@
  * Class which creates mapping between keys and a list of values.
  */
 @SuppressWarnings("unchecked")
-public class HashList extends HashMap {
+public class HashList extends HashMap<Object, Object> {
```

Comment:
Not sure how this is being used in the larger project, but would it make sense to replace this with generics - HashMap<K,V> as opposed to <Object, Object> ???